doc/commands.md

```diff
@@ -202,6 +202,26 @@ arguments on the left side of the `≔` symbol (similarly to a value declaration
 in OCaml). Some arguments can be declared as implicit by enclosing them in
 curly brackets.
 
+<!---------------------------------------------------------------------------->
+### `inductive`
+The command `inductive` is used to define an inductive type.
+For example:
+```
+inductive nat : TYPE ≔
+ | z    : nat
+ | succ : nat → nat
+```
+is equivalent to:
+
+```
+constant symbol nat  : TYPE
```

Comment:
Well, in fact, we should not declare the type as constant for now because otherwise we won't be able to add a rule `Nat -> \tau nat`.

Reply:
Yes exactly! So I can change it in the code?